test/unit/data/datatypes/test_qiime2.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+
+from galaxy.datatypes.qiime2 import strip_properties
+
+
+# Note: Not all the expressions here are completely valid types they are just
+# representative examples
+class TestStripProperties(unittest.TestCase):
```

Comment:
It would be good to add a tests for some "unkown future" binOp like `@` or really whatever. Just to make sure that passes through unscathed.

Reply:
Also add a test where you execute the helper twice.

Reply:
yep